src/peer/identity.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const PeerId = require('peer-id')
+const crypto = require('libp2p-crypto')
+
+const KEY_TYPE = 'RSA'  // change to ECC when possible
```

Comment:
State of ECC libs seems a bit unclear; in the browser we'd need to use something like [elliptic](https://github.com/indutny/elliptic), but there's probably a wrapper lib for some better tested code for node